core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -245,14 +245,14 @@ private[deploy] class Master(
       logError("Leadership has been revoked -- master shutting down.")
       System.exit(0)
 
-    case WorkerDecommission(id, workerRef) =>
-      logInfo("Recording worker %s decommissioning".format(id))
```

Comment:
We don't need to handle the MasterInStandby case anymore ? I guess all messages from worker to master should have this check, right ?

Reply:
Good catch!